po/de.po

```diff
@@ -5691,7 +5698,7 @@ msgstr "Kettenlademechanismus ersetzt Munitionsgurte"
 
 #: po/custom/fromJson.txt:317
 msgid "Chaingun Upgrade"
-msgstr "Verbesserung des Kettenfeuermechanismus"
+msgstr "Verbesserung des Kettenladers"
```

Comment:
Hast du eine Quelle dafür? (Nein, für meine Übersetzung habe ich auch keine, ich frage aus Neugier :P)

Reply:
Nein, habe ich nicht. Eine kurze Recherche bei [Wikipedia](https://de.wikipedia.org/wiki/Chain_Gun) hat auch ergeben, dass "Chaingun" auch eigentlich eine Waffe ist und kein Lademechanismus. Da dieses Upgrade aber die Feuerrate erhöht sollten wir das in der Übersetzung auch so darstellen. Da die Waffe über eine Kette geladen wird halte ich "Kettenlader" für am sinnvollsten.

Reply:
:heavy_check_mark: